libmachine/mcnutils/b2d.go

```diff
@@ -144,6 +144,12 @@ func (b *b2dReleaseGetter) getReleaseURL(apiURL string) (string, error) {
 	}
 
 	log.Infof("Latest release for %s/%s/%s is %s", host, org, repo, tag)
+	if tag == "v1.9.1" {
+		log.Warnf("Boot2Docker v1.9.1 has a known issue with AUFS. " +
```

Comment:
Some nitpicks:

- Can you please make "v1.9.1" a `const` in top level of the file named something like `aufsBugB2DVersion` and use that here?
- Why not use backticks for the multi-line strings instead and only have one `Warnf` call? And also why not make it a suggestion instead of a command?  Also, since you're not interpolating anything, please use `Warn` instead of `Warnf`.  i.e.:

```go
log.Warn(`
Boot2Docker v1.9.1 has a known issue with AUFS.
See here for more details: https://github.com/docker/docker/issues/18180
Consider specifying another storage driver (e.g. 'overlay') using '--engine-storage-driver' instead.
`)
```